pkg/gitops/config.go

```diff
@@ -38,10 +39,11 @@ func NewConfig() (config, error) {
 	if err := stepconf.Parse(&cfg); err != nil {
 		return config{}, fmt.Errorf("parse step config: %w", err)
 	}
-	cfg.Values = parseMap(cfg.RawValues)
-	return cfg, nil
+
+	return cfg, yaml.Unmarshal([]byte(cfg.RawValues), &cfg.Values)
```

Comment:
FYI @szabolcsgelencser we have an issue with templating as the param we want to pass contains a `:` (it is the image & tag together, coming from `build-env`). This, unfortunately breaks parsing, so that's why we're aiming to use the raw yaml string directly and then parsing that.

Reply:
Already mentioned but thanks! :)